homeassistant/core.py

```diff
@@ -1807,17 +1807,14 @@ def call(
         service_data: dict[str, Any] | None = None,
         blocking: bool = False,
         context: Context | None = None,
-        limit: float | None = SERVICE_CALL_LIMIT,
```

Comment:
We should remove the `SERVICE_CALL_LIMIT` constant from our code base (we can add it back for the tests).